frontend/src/store/summaryDeclaration.js

```diff
@@ -30,6 +30,74 @@ function getProgramYear(selectedGuid, programYearList) {
   return programYear;
 }
 
+/**
+ * Contact the various endpoints to summarize all the details that a client needs to know about any given facility in
+ * their summary declaration.
+ *
+ * @param {Object} facility - The facility to fill with details
+ */
+async function mapFacility(facility) {
+  const applicationStore = useApplicationStore();
+  const appStore = useAppStore();
+  let facilityLicenseResponse = undefined;
+  try {
+    facilityLicenseResponse = (
+      await ApiService.apiAxios.get(`${ApiRoutes.FACILITY}/${facility.facilityId}/licenseCategories`)
+    ).data;
+    facility.licenseCategories = parseLicenseCategories(facilityLicenseResponse);
+  } catch {
+    console.log('error, unable to get childcare category for provider: ', facility.facilityId);
+  }
+
+  // check for opt out - no need for more calls if opt-out
+  if (facility.ccfri?.ccfriId && facility.ccfri?.ccfriOptInStatus == 1) {
+    const ccfriPromises = [ApiService.apiAxios.get(ApiRoutes.CCFRIFACILITY + '/' + facility.ccfri.ccfriId)];
```

Comment:
Can we replace the string concatenation with template literals (see line 45) while we're updating this code?

Reply:
Done :)